cluster/userdata-worker.yaml

```diff
@@ -232,11 +232,11 @@ write_files:
       TOOLBOX_DOCKER_IMAGE=registry.opensource.zalan.do/teapot/microscope
       TOOLBOX_DOCKER_TAG=v0.0.4
 
-  - path: /etc/systemd/timesynced.conf
+  - path: /etc/systemd/timesyncd.conf
     owner: root
     content: |
       [Time]
-      NTP=0.amazon.pool.ntp.org 1.amazon.pool.ntp.org 2.amazon.pool.ntp.org 3.amazon.pool.ntp.org
+
```

Comment:
Looks like it's missing here.

Reply:
Oops. :( My editor ate it and I didn't review the diff, sorry about that.